packages/cli/src/commands/env/pull.ts

```diff
@@ -81,7 +81,7 @@ export default async function pull(
   }
 
   output.print(
-    `Downloading Development Environment Variables for Project ${chalk.bold(
+    `Downloading ${environment} Environment Variables for Project ${chalk.bold(
```

Comment:
Note that this will be lowercase target (i.e. "development" rather then "Development") since it's the enum value. I think we either need to capitalize to have the same format or add quotations around the enum value.